package.json

```diff
@@ -29,13 +29,10 @@
 	],
 	"type": "module",
 	"exports": {
-		".": [
-			{
-				"import": "./dist/index.mjs",
-				"require": "./dist/index.cjs"
-			},
-			"./dist/index.cjs"
-		]
```

Comment:
Oh, which variation of this were folks in #277 testing?

Reply:
Yes they tested it with the array but it seems that doesn't have any effect if we already explicitly mention `import` and `require`

Reply:
Eh, I guess it shouldn't matter if the entire issue was that eslint was ignoring this field.